### DISCUSSION TITLE
Creating issue from task list should use the initial issue settings

### DISCUSSION THREAD

**Discussion by DarkSkyT9A** [community/community#4690] [Projects and Issues]
When creating an issue from a task list, usually the properties of the task list (i.e. labels, projects, milestones) also apply to the newly generated issue. These should be automatically filled in or at least the option to do so should be there (i.e. dropdown with options: blank, parent, template)

**Comment by vykhrystiuk** [comment]
Agreed, I also find it useful, it will simplify the process of creating issues from the list and will prevent you from missing the project and milestone by accident when creating issues.

**Comment by st-pasha** [comment]
Note to maintainers: this was also requested before in #4544 and #4330.
